src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -2677,9 +2630,13 @@ internal bool LoadModuleManifestData(ExternalScriptInfo scriptInfo, ManifestProc
                     }
                 }
 
+                // We have to further analyze the module if any wildcard characters are used.
                 var needToAnalyzeScriptModules = usedWildcard;
```

Comment:
Can this be changed to `bool`? Just one extra character

Reply:
Sure. Fixed.